mp/src/game/shared/momentum/mom_rocket.cpp

```diff
@@ -43,6 +49,11 @@ END_NETWORK_TABLE();
 LINK_ENTITY_TO_CLASS(momentum_rocket, CMomRocket);
 PRECACHE_WEAPON_REGISTER(momentum_rocket);
 
+static MAKE_TOGGLE_CONVAR(mom_rj_rocketmodel, "0", FCVAR_ARCHIVE | FCVAR_CLIENTDLL,
```

Comment:
Same with the view model, `mom_rj_use_tf_rocketmodel` is much more clear in intent here.